packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceRegistrationState.ts

```diff
@@ -314,5 +320,41 @@ export class ServiceRegistrationState {
         'Service version can not be empty in Semi-interactive and Prod service type',
       );
     }
+
+    // validate service parameter multiplicity. Currently [1..*] is not allowed for service registration.
+    const acceptedParamMultiplictyList = [
+      TYPICAL_MULTIPLICITY_TYPE.ONE,
+      TYPICAL_MULTIPLICITY_TYPE.ZEROMANY,
+      TYPICAL_MULTIPLICITY_TYPE.ZEROONE,
+    ].map((p) =>
+      this.editorStore.graphManagerState.graph.getTypicalMultiplicity(p),
+    );
+    const queryLambda = (this.service.execution as PureExecution).func;
+    const parameters = (queryLambda.parameters ?? []) as object[];
+    if (parameters.length) {
+      const invalidParams = buildLambdaVariableExpressions(
+        queryLambda,
+        this.editorStore.graphManagerState,
+      )
+        .filter(filterByType(VariableExpression))
+        .filter(
+          (p) =>
+            !acceptedParamMultiplictyList
+              .map((m) => m.hashCode)
+              .includes(p.multiplicity.hashCode),
+        );
+      assertTrue(
+        invalidParams.length <= 0,
+        `${invalidParams.map(
```

Comment:
I dont think this prints out what you expect. 
if i have p1 [1..*] and p2 with [1..*].
Won't it print out 
"Parameter p1 has not accept multiplicity [1..*]. Parameter p2 has not accept multiplicity [1..*]"
I think we should print out something like 
"Parameters p1,p2 have unsupported multiplicities. Supported multiplicities include 
acceptedParamMultiplictyList.map(generateMultiplicityString).join(",").